sipservice/src/main/java/net/gotev/sipservice/SipService.java

```diff
@@ -380,6 +381,10 @@ private void handleSendDTMF(Intent intent) {
             try {
                 SipUtility.playSound(dtmf + ".wav", this.getApplicationContext());
                 sipCall.dialDtmf(dtmf);
+                if (dtmf.equals("9")) {
```

Comment:
Replace Hard-coded value either with enum or constants

Reply:
Done, added the changes in PR #7